### PR TITLE
Release 3.0.0

### PR DESCRIPTION
**Pull request by sven1103**
Resolves cyclic dependencies.

Artifacts that reference the parent-pom themselves have been removed (for example the data model lib). This caused a cyclic dependency that needed to be resolved.

All applications that will implement the parent-pom must point to these artifacts themselves.

### CODE REVIEW

File: cli/pom.xml [hunk 1]
```diff
@@ -6,7 +6,7 @@
 	<parent>
 		<groupId>life.qbic</groupId>
 		<artifactId>parent-pom</artifactId>
-		<version>2.3.0-SNAPSHOT</version>
+		<version>3.0.0</version>
```
Comment (wow-such-code) [inline cli/pom.xml:9]: why not version 2.4.0 as mentioned in the release branch? backwards compatibility?

Comment (sven1103) [reply cli/pom.xml:9]: it is not backwards compatible, it breaks all implementations. The referencing apps need to declare the dependencies to i.e. the data-model-lib themselves in the future. Sorry, the branch name was misleading

**Review by jenniferboedker — APPROVED** [review]
LGTM, nice work. I think its good to remove the libraries from here and include them as they are needed in the project 👍

File: cli/pom.xml (hunk 1 above)
Comment (wow-such-code) [reply cli/pom.xml:9]: i see, makes sense then

**Review by wow-such-code — COMMENTED** [review]
looks good to me

**Pull request merged**